user.go

```diff
@@ -2,6 +2,27 @@ package discordgo
 
 import "strings"
 
+// UserFlags is the flags of "user" (see UserFlags* consts)
+// https://discord.com/developers/docs/resources/user#user-object-user-flags
+type UserFlags int
+
+// Valid UserFlags values
+const (
+	UserFlagDiscordEmployee      UserFlags = 1 << iota
+	UserFlagDiscordPartner                 = 1 << iota
```

Comment:
The `= 1 << iota` bit is only necessary on the first line; because the flags are each explicitly specified in the docs I’d be fine if each line has the value, but if you’d rather just use `iota` then only the first is necessary.

Reply:
Oh well, actually did not know that the bitshitf operation with `itoa` is also adopted to the other values.